ext/date/php_date.stub.php

```diff
@@ -673,19 +673,40 @@ class DatePeriod implements IteratorAggregate
     /** @cvalue PHP_DATE_PERIOD_INCLUDE_END_DATE */
     public const int INCLUDE_END_DATE = UNKNOWN;
 
-    /** @readonly */
+    /**
+     * @readonly
+     * @virtual
+     */
```

Comment:
Ugh, this is why I was against changing it to a one line attribute style the other time this got touched.

Reply:
Yeah, I can see your point :)